### DISCUSSION TITLE
How to set up dialog window?

### DISCUSSION THREAD

**Discussion by Ziggy5454** [palexdev/MaterialFX#189] [Archived]
Is there any quick way to get a dialog box to show without implementing it through scenebuilder? Or any guides out there where I could learn more about how to do this specifically for MaterialFX? I was using basic java jframe and was just able to use one line of quick code, been messing around with MaterialFX's dialog panes but having issues. Awesome work by the way!

**Comment by palexdev** [comment]
In recents versions of MaterialFX there's a generic dialog which can show a text and a title
It can be easily built using the MFXGenericDialogBuilder class, also you can convert it to a Stage dialog using the MFXStageDialogBuilder class. Check both the src code and javadocs to learn more about them
